README.md

```diff
@@ -28,9 +28,32 @@ of your proposal once assigned, e.g. `assets/aex-1/image.png`.
 
 Everything beyond step 4 is governed by [AEX-1](https://github.com/aeternity/AEXs/blob/master/AEXS/aex-1.md).
 
+## Status terms
+
+
+* Draft - an AEX that is open for consideration and is undergoing rapid iteration and changes.
```

Comment:
This list is not complete. Could you please make it reflect AEX-1 or remove it?

Reply:
Update README